### PR TITLE
changes to lexorin and sulfonal recipes

### PR DESCRIPTION
**Pull request by That0nePerson**
## About The Pull Request
Lexorin has been talked about as one of the best toxins for a while, and it has a piss-easy recipe, only requiring 3 basic chems. Sulfonal is objectively much worse than lexorin, yet has a much, much harder recipe. After some discussion on the discord, stormcaster has recommended these changes to the recipes. Sulfonal now takes perfluoro instead of acetone, and lexorin takes sulfonal as a 4th ingredient and giving 4u.

## Why It's Good For The Game
See above, lexorin has been super easy to make and OP, while sulfonal has been pretty bad, with lexorin being basically an upgrade. This balances things a bit, making the recipe for lexorin take sulfonal and making it sort of a step-up from sulfonal.

## Changelog
:cl:
balance: sulfonal now takes perfluorodecalin instead of acetone
balance: lexorin now takes sulfonal, gives 4u.
/:cl:

### CODE REVIEW

**Comment by eLeCtrOssSnake** [comment]
If you try to get the idea of how these chems work, it's pretty obvious that sulfonal is just a toxin, that is just charcoal defeated and lexorin being a really fast toxin that suffocates, but the thing is that it requires plasma, so that kinda makes sense why the other reagents are so easy. In conclusion, this pull request won't balance anything nor change it, so I would rate it as useless and unneeded.

**Comment by That0nePerson** [comment]
plasma is easily available from all department lathes at maximum 30 minutes into the round, earlier if
 mining is good

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	name = "Lexorin"
 	id = /datum/reagent/toxin/lexorin
 	results = list(/datum/reagent/toxin/lexorin = 3)
```
Comment (zeskorion) [inline code/modules/reagents/chemistry/recipes/toxins.dm]: change to 4

**Review by zeskorion — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/recipes/toxins.dm (hunk 1 above)
Comment (That0nePerson) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: done

**Comment by Chessus** [comment]
You start with plasma in chemistry on meta station also. You've got a access to a potent  toxin from roundstart.

**Comment by Isy232** [comment]
This is good. 15u of lexorin does 187.5 oxyloss damage while also stopping breathing. If not given any help it will kill you quickly. You will also be knocked out partway through because of oxygen deprevation, meaning you have little time to react.

Lexorin is the deadliest poison mixable in the game and yet it does not require anything but plasma and 2 other base chems to make.

I've personally never seen one, but a lexorin massacre wouldn't be horribly difficult to pull off.

This is a good nerf.

**Comment by zeskorion** [comment]
> This is good. 15u of lexorin does 187.5 oxyloss damage while also stopping breathing. If not given any help it will kill you quickly. You will also be knocked out partway through because of oxygen deprevation, meaning you have little time to react.
> 
> Lexorin is the deadliest poison mixable in the game and yet it does not require anything but plasma and 2 other base chems to make.
> 
> I've personally never seen one, but a lexorin massacre wouldn't be horribly difficult to pull off.
> 
> This is a good nerf.

plasmaman mutation toxin can be deadlier

**Review by zeskorion — APPROVED** [review]

**Comment by Isy232** [comment]
> plasmaman mutation toxin can be deadlier

Except that requires black slimes from xenobio and is a lot more difficult to mass produce.

**Pull request merged**